SITnett/templates/meny.html

```diff
@@ -70,14 +70,13 @@
 			<div class="footer-section">
 				<p><a href="{% url 'kontakt' %}" class="text-strong">Kontakt oss</a></p>
 				<p> Studentersamfundet <br>
-					Elgesetergate 1 <br> 
+					Elgesetergate 1 <br>
```

Comment:
Fikse denne når vi først er i gang?
```suggestion
					Elgeseter gate 1 <br>
```